### PR TITLE
doc: added info to bitcoin.conf doc

### PR DESCRIPTION
**Pull request by bliotti**
Should probably be explicitly stated to not make modifications to the conf file while daemon is running. ref #11586

For example, if rpc credentials are modified while bitcoind is running, `bitcoin-cli stop` is unable to stop bitcoind until the original credentials are restored in `bitcoin.conf`

### CODE REVIEW

File: doc/bitcoin-conf.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ The configuration file is used by `bitcoind`, `bitcoin-qt` and `bitcoin-cli`.
 
 All command-line options (except for `-?`, `-help`, `-version` and `-conf`) may be specified in a configuration file, and all configuration file options (except for `includeconf`) may also be specified on the command line. Command-line options override values set in the configuration file and configuration file options override values set in the GUI.
 
+The configuration file should not be modified while `bitcoind` is running.
```
Comment (hebasto) [inline doc/bitcoin-conf.md:7]:
```suggestion
The configuration file should not be modified while `bitcoind` or `bitcoin-qt` is running.
```

**Review by hebasto — COMMENTED** [review]
Concept ACK.

**Comment by Zero-1729** [comment]
Concept ACK (esp with @hebasto's suggestion)

**Comment by sipa** [comment]
I'm not sure that's the advice we should give. There is no problem with modifying the config file while Bitcoin Core is running per se; it's just that changes won't take effect until you restart.

**Comment by Zero-1729** [comment]
To @sipa's point, if you are going add a warning it should probably be to notify users that the changes take effect only after restart. `bitcoin-qt` already warns the users when the modify the configuration file while running. 

So I suppose maybe rewording it to something like:

```
Changes to the configuration file while `bitcoind` or `bitcoin-qt` is running only takes effect after reset.
```

**Comment by PapauloGamerOfc** [comment]
Concept ACK. (I found this Pull Request in a Brazilian live).

**Comment by brunoerg** [comment]
> To @sipa's point, if you are going to add a warning it should probably be to notify users that the changes take effect only after restart. `bitcoin-qt` already warns the users about it when they modify the configuration file.
> 
> So I suppose maybe rewording it to something like:
> 
> ```
> Changes to the configuration file while `bitcoind` or `bitcoin-qt` is running only takes effect after reset.
> ```

I think `restarting` instead of `reset` would be better.

**Comment by jonatack** [comment]
and s/takes/take/

**Comment by Zero-1729** [comment]
Thanks @brunoerg and @jonatack for the catch! It should have been `restarting` instead of `reset`. I also think `take` fits better, grammatically speaking.

So it should read:

```
Changes to the configuration file while `bitcoind` or `bitcoin-qt` is running only take effect after restarting.
```

**Comment by Meru852** [comment]
How to find other wallet address privatekey use the bitcoin-qt?

Pada Senin, 05 Juli 2021, Abubakar Nur Khalil ***@***.***>
menulis:

**Review by Zero-1729 — APPROVED** [review]
ACK fa84caebc7b647d9483262b6634bfe6b02b90a69

**Review by theStack — APPROVED** [review]
LGTM
ACK fa84caebc7b647d9483262b6634bfe6b02b90a69 🗄️

**Comment by rajarshimaitra** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/22393/commits/fa84caebc7b647d9483262b6634bfe6b02b90a69
